autokeras/blocks/basic.py

```diff
@@ -154,6 +154,56 @@ def build(self, hp, inputs=None):
         return output_node
 
 
+class Attention(block_module.Block):
```

Comment:
In what end-to-end workflow is this intended to be used?

Reply:
We don't have an end-to-end workflow for it now. We implemented it by accident. I think it is potentially use by other blocks to enlarge the search space.